network-api/networkapi/templates/pages/base.html

```diff
@@ -16,6 +16,12 @@
   </head>
   <body id="view-{% block bodyID %}{% endblock %}">
 
+    {% if user.is_authenticated %}
+    <div class="authenticated-page-edit" style="position: fixed; bottom:0; background: teal; padding: 0.25em; z-index:1000">
+      <a style="color: white" href="/cms/pages/{{page.id}}/">Edit</a>
```

Comment:
```suggestion
      <a style="color: white" href="/cms/pages/{{page.id}}/edit">Edit</a>
```

Reply:
See original comment- this was intentional

Reply:
This does make it fairly cumbersome to edit pages, through - if we want to make sure people without edit rights still see something useful, we should be able to test for that to make sure the template renders the right link up front?